editor/editor_audio_buses.cpp

```diff
@@ -644,18 +651,43 @@ Variant EditorAudioBus::get_drag_data_fw(const Point2 &p_point, Control *p_from)
 
 bool EditorAudioBus::can_drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from) const {
 	Dictionary d = p_data;
-	if (!d.has("type") || String(d["type"]) != "audio_bus_effect") {
+
+	if (!d.has("type")) {
 		return false;
 	}
 
-	TreeItem *item = effects->get_item_at_position(p_point);
-	if (!item) {
-		return false;
+	if (String(d["type"]) == "audio_bus_effect") {
```

Comment:
I would use:
`"type" = "resource"`
and
`"resource" = the effect`

Instead of creating a new dnd

Reply:
I'm not sure that I understand what you mean, could you please elaborate ?

Reply:
It already existed here, so it doesn't need to be changed in this PR.